pkg/apis/core/v1/helper/helpers.go

```diff
@@ -413,6 +413,33 @@ func TolerationsTolerateTaintsWithFilter(tolerations []v1.Toleration, taints []v
 	return true
 }
 
+// FindMatchingUntoleratedTaint checks if the given tolerations tolerates
+// all the filtered taints, and returns the first taint without a toleration
+func FindMatchingUntoleratedTaint(taints []v1.Taint, tolerations []v1.Toleration, applyFilter taintsFilterFunc) *v1.Taint {
```

Comment:
s/applyFilter/filter/

Reply:
Actually "inclusionFilter" is clearer-- you'll include taints for which the filter returns true.

Reply:
changed